packages/pangea-node-sdk/src/utils/utils.ts

```diff
@@ -78,13 +78,16 @@ export function hashSHA512(data: string): string {
   return sha512.finalize().toString();
 }
 
-export function hashNTLM(password: string): string {
-  // Calculate the MD4 hash
-  const md4Hash = crypto.createHash("md4");
-  md4Hash.update(Buffer.from(password, "utf16le"));
+export async function hashNTLM(password: string): Promise<string> {
```

Comment:
This is a breaking change to a public API so this will have to be a major release.